exercises/concept/squeaky-clean/src/test/java/SqueakyCleanTest.java

```diff
@@ -43,50 +43,50 @@ public void leading_and_trailing_spaces() {
 
     @Test
     @Tag("task:2")
-    @DisplayName("The clean method replaces control characters with CTRL")
-    public void ctrl() {
-        assertThat(SqueakyClean.clean("my\0\r\u007FId")).isEqualTo("myCTRLCTRLCTRLId");
+    @DisplayName("The clean method converts kebab to camel case after removing a dash")
+    public void kebab_to_camel_case() {
+        assertThat(SqueakyClean.clean("à-ḃç")).isEqualTo("àḂç");
     }
 
     @Test
-    @Tag("task:4")
-    @DisplayName("The clean method returns an empty string when invoked on a string with no letters")
-    public void string_with_no_letters() {
-        assertThat(SqueakyClean.clean("\uD83D\uDE00\uD83D\uDE00\uD83D\uDE00")).isEmpty();
+    @Tag("task:2")
+    @DisplayName("The clean method returns a string in camel case after removing a dash and replacing the whitespace")
+    public void kebab_to_camel_case_and_number() {
+        assertThat(SqueakyClean.clean("a-C ")).isEqualTo("aC_");
     }
 
     @Test
-    @Tag("task:3")
-    @DisplayName("The clean method converts kebab to camel case after removing a dash")
-    public void kebab_to_camel_case() {
-        assertThat(SqueakyClean.clean("à-ḃç")).isEqualTo("àḂç");
+    @Tag("task:2")
+    @DisplayName("The clean method returns a string in camel case after removing a dash and replaces whitespaces")
```

Comment:
The display name of this test and the one before are practically the same. Can we make them a little more distinguishable?

Reply:
Done!